listenbrainz/db/user_timeline_event.py

```diff
@@ -170,6 +183,40 @@ def get_recording_recommendation_events_for_feed(user_ids: List[int], min_ts: in
         return [UserTimelineEvent(**row) for row in result.fetchall()]
 
 
+def get_personal_recommendation_events_for_feed(user_id: int, min_ts: int, max_ts: int, count: int) -> List[UserTimelineEvent]:
+    """ Gets a list of personal_recording_recommendation events for specified users.
+
+    user_ids is a tuple of user row IDs.
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT user_timeline_event.id
```

Comment:
I reformatted the query myself to make a bit easier to read. Hope you don't mind.